examples/ola-client.cpp

```diff
@@ -278,6 +279,12 @@ void DisplayDevices(SelectServer *ss,
   ss->Terminate();
 }
 
+void SetPluginStateComplete(SelectServer *ss, const Result &result) {
```

Comment:
We should just merge all of these into a HandleAck(..) but that can be done later.